.github/workflows/deploy.yml

```diff
@@ -11,14 +11,16 @@ jobs:
     - uses: actions/checkout@v1
 
     - name: create a deployment
-      uses: npm/action-deploy@v2
+      uses: npm/action-deploy@v2.1.0
       id: create-deployment
       with:
         type: create
         token: ${{github.token}}
         logs: https://github.com/npm/action-deploy/actions?query=workflow%3Adeploy
         environment_url: https://npmjs.com
         job_status: ${{job.status}}
+        slack_token: ${{secrets.SLACK_NPM_ROBOT}}
```

Comment:
The name for this variable in `etcd` is `NPM_ROBOT_SLACK_TOKEN`. Probably keep it consistent, since one day we would switch from `etcd` to shared org secrets. 🤔

Reply:
Makes sense, will fix that.